STM32F405RGTx/Core/Inc/svpwm_thread.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "VHZ.h"
+#include "main.h"
+#include "util.hpp"
+
+#ifndef INC_SVPWM_H_
+#define INC_SVPWM_H_
+
+class SVPWM{
+    public:
+        static float  ModIndex;
+		static float  Freq;
+        static float  SwitchFreq;
+        static float  Alpha;

```

Comment:
I think many of these variables can be private (aside from stuff like modindex and frequency)